src/components/Tooltip/Tooltip.tsx

```diff
@@ -328,6 +334,7 @@ const Tooltip = ({
                 className="p-tooltip__message"
                 ref={messageRef}
                 id={tooltipId}
+                style={{ zIndex: zIndex }}
```

Comment:
What would happen if someone passes invalid value here?

Reply:
I just tested, the invalid value would be ignored. But maybe, as you mentioned in the other comment, this risk can be avoided by setting the `zIndex` prop's type to `number`. I will change that.